__tests__/test.sh

```diff
@@ -0,0 +1,13 @@
+PROJECT_ROOT=$(dirname $0)/../
```

Comment:
Could you rewrite this file in JavaScript or TypeScript for Windows?

Reply:
Hm, let's rewrite this in another issue/PR.

Reply:
I made issue #2